circuits/cpp/src/aztec3/circuits/kernel/private/common.cpp

```diff
@@ -343,7 +343,9 @@ void common_contract_logic(DummyBuilder& builder,
     const auto& storage_contract_address = private_call_public_inputs.call_context.storage_contract_address;
     const auto& portal_contract_address = private_call.portal_contract_address;
 
-    const auto private_call_vk_hash = stdlib::recursion::verification_key<CT::bn254>::hash_native(private_call.vk);
+    // TODO(Alvaro) VKs are mocked out for now
```

Comment:
Do we have an issue for this?

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/3062

Reply:
Added the issue number to the TODOs